config/vufind/export.ini

```diff
@@ -71,4 +71,5 @@ headers[] = "Content-type: application/x-bibtex; charset=utf-8"
 
 [RIS]
 requiredMethods[] = getTitle
-headers[] = "Content-type: application/x-research-info-systems; charset=utf-8"
\ No newline at end of file
+headers[] = "Content-type: application/x-research-info-systems; charset=utf-8"
+label = Citavi/RIS
```

Comment:
It does make sense as any librarian in Germany knows Citavi but nobody knows RIS.